source/includes/table-proxy-env-variables.rst

```diff
@@ -0,0 +1,82 @@
+.. list-table::
+   :header-rows: 1
+   :widths: 20 35 45
+
+   * - Variable
+     - Description
+     - Example
+
+   * - ``MONGODB_PROXY``
+
+     - Proxy connections to ``mongodb://`` and ``mongodb+srv://`` URLs, such 
+       as database clusters.
+
+     - The following example sets the ``MONGODB_PROXY`` environment variable to
+       proxy all MongoDB connections through a CONNECT proxy located at 
+       ``example.com:8080`` with TLS enabled.
+
+       .. code-block:: sh 
+          :copyable: false 
+          
+          export MONGODB_PROXY=https://example.com:8080
+
+   * - ``HTTP_PROXY``
+
+     - Proxy connections to ``http://`` URLs. HTTP connections primarily relate 
```

Comment:
[suggestion]

Consider simplifying language a bit here. Spitballing: "HTTP connections are mostly used for OIDC authentication." or simliar.

Reply:
Updated, thanks for the suggestion!